models_evaluation.py

```diff
@@ -0,0 +1,176 @@
+from os import listdir
+from os.path import isfile, join
+import sys
+import argparse
+import logging
+import pathlib
+
+import gensim
+import pandas as pd
+import itertools
+
+logger = logging.getLogger("fasttext")
+
+
+def questions_reading(analogies):
+    """
+    Reads all type of questions to dict.
+    :param analogies: string
+    :return: dict
+    """
+    analogy_dict = dict()
+    with open(analogies, "r") as analogy_f:
+        key = None
+        for line in analogy_f:
+            if line in ['\n', '\r\n']:
```

Comment:
`if not line.strip():`
looks like a more robust solution (as it will also cover "empty" lines with spaces)